### PR TITLE
Using pre-built code for testing.

### PR DESCRIPTION
**Pull request by yurydelendik**
(... in hope to fix Chrome failures on test servers)

- Unit test will use pre-built worker code when API code is testing
- Ref tests will use pre-built main and worker code

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1055,34 +1055,34 @@ gulp.task('publish', ['generic'], function (done) {
...
   return createTestSource('browser');
 });
 
-gulp.task('unittest', function () {
+gulp.task('unittest', ['generic'], function () {
   return createTestSource('unit');
 });
 
-gulp.task('fonttest', function () {
+gulp.task('fonttest', ['generic'], function () {
```
Comment (Snuffleupagus) [inline gulpfile.js]: Unless you also update `test/font/jasmine-boot.js`, I don't think this change is necessary!?

Comment (yurydelendik) [inline gulpfile.js]: fixed

**Comment by yurydelendik** [comment]
Let's update ref images.

/botio-windows makeref

**Comment by Snuffleupagus** [comment]
Hmm, looking at the end of the output from the Windows bot in either in https://github.com/mozilla/pdf.js/pull/8459#issuecomment-304925517 (see http://54.215.176.217:8877/87ca7b61808c5a3/output.txt) or https://github.com/mozilla/pdf.js/pull/8459#issuecomment-304938909 (see http://54.215.176.217:8877/34da9c465d2c659/output.txt), it seems that not all test actually ran in *Firefox*!

For Firefox, in the first case https://github.com/mozilla/pdf.js/blob/master/test/test_manifest.json#L3409 is the last test in the log; in the second case https://github.com/mozilla/pdf.js/blob/master/test/test_manifest.json#L3395 is the last test.

Are the logs just incomplete, or didn't all tests get run in Firefox on the Windows bot?

**Comment by yurydelendik** [comment]
I think it's just hitting 200kb maxBuffer limit (https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback) -- it shall be fine when "no-reference" messages will go away.

**Comment by Snuffleupagus** [comment]
Landed as part of PR #8284 (obviously with the correct author info).

**Pull request closed without merging**